### DISCUSSION TITLE
How is the floating expansion menu in the upper left corner implemented?

### DISCUSSION THREAD

**Discussion by allen0234** [web-infra-dev/rspress#633] [Q&A] [answered]
看起来很cool👍
![image](https://github.com/web-infra-dev/rspress/assets/49856281/d17b2525-7799-45ad-9eff-cea4b0e1ed5d)

**Comment by sanyuan0704** [comment, marked as answer]
通过自定义主题的 beforeNavTitle 的插槽插入的，组件封装在 rsfamily-nav-icon 包里面 https://github.com/web-infra-dev/rspress/blob/main/packages/document/theme/index.tsx#L28
